.gitignore

```diff
@@ -2,4 +2,5 @@
 **/.DS_Store
 **/.vscode/**
 **/bin/**
+**/src/**
```

Comment:
Ignora eventuali file oggetto che verranno salvati in `other_examples/src`. Il comando `make clean` li elimina, ma qual'ora mi dimenticassi di farlo, questa guardia impedisce il caricamento a ogni modo.